src/chigame/games/views.py

```diff
@@ -175,17 +190,34 @@ class TournamentUpdateView(UpdateView):
         "rules",
         "draw_rules",
         "num_winner",
-        "matches",
```

Comment:
Your PR description says staff members should be able to add/remove players, but you removed players and matches, so it doesn't allow for this functionality.